pkg/kubectl/resource_printer.go

```diff
@@ -446,9 +445,9 @@ func (h *HumanReadablePrinter) HandledResources() []string {
 	return keys
 }
 
-func (h *HumanReadablePrinter) FinishPrint(output io.Writer, res string) error {
-	if !h.options.NoHeaders && !h.options.ShowAll && h.hiddenObjNum > 0 {
-		_, err := fmt.Fprintf(output, "  info: %d completed object(s) was(were) not shown in %s list. Pass --show-all to see all objects.\n\n", h.hiddenObjNum, res)
+func (h *HumanReadablePrinter) FinishPrint(output io.Writer, res string, hiddenObjNum int) error {
```

Comment:
Yeah, definitely agree. I will look into making this part of the filter, rather than leaving it as part of the printer